recipes/pixman/all/conanfile.py

```diff
@@ -92,22 +106,20 @@ def build(self):
             autotools.make(target="pixman")
 
     def package(self):
+        self.copy(os.path.join(self._source_subfolder, "COPYING"), dst="licenses")
         if self.settings.compiler == "Visual Studio":
             self.copy(pattern="*.lib", dst="lib", keep_path=False)
             self.copy(pattern="*{}pixman.h".format(os.sep), dst=self._includedir, keep_path=False)
             self.copy(pattern="*{}pixman-version.h".format(os.sep), dst=self._includedir, keep_path=False)
         else:
             autotools = self._configure_autotools()
             autotools.install()
-            tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
-            la = os.path.join(self.package_folder, "lib", "libpixman-1.la")
-            if os.path.isfile(la):
-                os.unlink(la)
-        self.copy(os.path.join(self._source_subfolder, 'COPYING'), dst='licenses')
+            tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
-        self.cpp_info.includedirs = [self._includedir]
-        self.cpp_info.names['pkg_config'] = 'pixman-1'
-        if self.settings.os == "Linux":
+        self.cpp_info.includedirs.append(self._includedir)
```

Comment:
append? What is the content of pkgconfig file?

Reply:
This gives the same behavior as when installing in a linux prefix, e.g. `/usr`.
You can then include the headers using `pixman/pixman.h` or `pixman.h`.

Reply:
you mean `pixman-1/pixman.h`

Reply:
Yes, I was just giving an example to make my point. I didn't look at the .pc file.